packages/utils/src/fallbackSubgraph.ts

```
@@ -175,116 +225,158 @@ export class FallbackSubgraph<T extends SubgraphSdk> {
-    await Promise.all(
-      this.subgraphs.map(async (subgraph, index) => {
-        const errors: any[] = [];
-        let latestBlock = _latestBlock;
-        let syncedBlock: number | undefined;
-        // First, try to use the subgraph health API.
-        try {
-          const health = await withRetries(async () => await getSubgraphHealth(subgraph.record.name, subgraph.url));
-          if (health && health.latestBlock && health.chainHeadBlock) {
-            // Sanity check: make sure the values are valid numbers.
-            latestBlock = typeof health.latestBlock === "number" ? health.latestBlock : latestBlock;
-            syncedBlock = typeof health.chainHeadBlock === "number" ? health.chainHeadBlock : undefined;
-          } else if (health && health.fatalError) {
-            throw new NxtpError("Subgraph health query response had fatal error present.", {
-              health,
-            });
-          } else if (!health || !health.latestBlock || !health.chainHeadBlock) {
-            throw new NxtpError("Subgraph health query returned invalid value.", {
-              health,
-            });
-          }
-        } catch (e) {
-          errors.push(e);
-        }
-
-        // Assuming we have the latestBlock passed in, and the above health query failed, we should
-        // use the typical GetBlockNumber next.
-        if (latestBlock && !syncedBlock) {
-          try {
-            const { _meta } = await withRetries(async () => await subgraph.client.GetBlockNumber());
-            syncedBlock = _meta && _meta.block && _meta.block.number ? _meta.block.number : 0;
-          } catch (e) {
-            errors.push(e);
-          }
-        }
+      // Target this chain's endpoint.
+      const endpointUrl = endpoint.concat(`?chainId=${this.chainId}`);
+      let response: AxiosResponse<SubgraphHealth[]> | undefined = undefined;
+      try {
+        response = await axios.get(endpointUrl);
+      } catch (e) {}
```

Swallowing the error is likely not a good idea (it should at least have the `message` logged at a debug level).

Also, would we explicitly not want to retry this request as well?

---

We could retry this.

Also, I'm not going to log it here, but I'll attach the error to the subgraph sync record. The real reason we ignore the error is because it basically just indicates the health endpoint is down / currently not supported - meaning we should resort to using the getBlockNumber method.